x-pack/packages/ml/landing_links/src/landing_links.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
```

Comment:
A note before you continue on this path: all the files that you have copied in the `src` folder look like generic components. 
We probably want to have a generic package for them (e.g. `packages/landing_pages`) and not limited `ml`. If tomorrow we want to do the same with `dashboards` we probably don't want to duplicate all those components over there 😊

Reply:
Ok cool! I can create a generic `packages/landing_pages` package. Do you have any recommendation regarding where to import `constants`, `context`, `types` from? These are currently defined in the `kbn/security-solution-navigation` package and [here](https://github.com/elastic/kibana/pull/193821/commits/7c67d0e48ef01ea559817d1a1879cf0ddf88c40e) I load them from the security navigation package.

Reply:
The `SecurityPageName` is not important for the test. The generic component should have props to customize it and probably `SecurityPageName` would be passed there.